Tempura/Sources/Navigation/Navigator.swift

```diff
@@ -180,6 +180,11 @@ public class Navigator {
   private var window: UIWindow! // swiftlint:disable:this implicitly_unwrapped_optional
   private let routableProvider: RoutableProvider
 
+  /// The indentifiers of the routables in the visible hierarchy.
+  public var currentRoutableIdentifiers: Route {
+    self.routableProvider.currentRoutable().map(\.routeIdentifier)
+  }
+
```

Comment:
Shouldn't we expose it also (or maybe only) from the `NavigatorWitness`? Otherwise you won't be able to test the logic using this.